packages/client/src/products/types/productListingFacets.types.ts

```diff
@@ -0,0 +1,26 @@
+type ProductListingFacetValue = {
+  value: number;
+  valueUpperBound: number;
+  description: string;
+  slug: string;
+  url: string;
+  parentId: number;
+  _isDisabled: boolean;
+  _isActive: boolean;
+  groupsOn: number;
+  count: number;
+};
+
+export type ProductListingFacet = {
```

Comment:
What is the difference between this type and `FacetGroup` type that represents the facets that comes on the get product listing response? I think they are the same thing. Can you check, please?